README.md

```diff
@@ -203,10 +203,14 @@ The pattern should something like "base-%03d.html"
 ### hocr-wordfreq
 
 ```
-hocr-wordfreq file.html
+hocr-wordfreq [-h] [-i] [-n MAX] [file.html]
```

Comment:
That is one solution. I guess the question is how long we want the README to be useful. Should the README really cover everything and is it just the starting point. Anyway, we can do this (in both ways) later.